answer/kazutaka/css/2/02css.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+
+    <style>
+        .all{
+    background-color: rgb(187, 176, 176);
+    padding: 20px;
+    }
```

Comment:
```suggestion
    .all{
        background-color: rgb(187, 176, 176);
        padding: 20px;
    }
```

インデントは↑みたいにするようにしましょうか